dao-erc20/src/erc20.rs

```diff
@@ -125,6 +118,38 @@ impl ERC20 {
             value: allowance - amount,
         });
     }
+
+    pub fn mint(&mut self, recipient: &Address, amount: U256) {
```

Comment:
Why do all the other methods take an address, but `mint()` and `burn()` take a reference?

Reply:
Because this function should never be directly exposed via `delegate` so it's build to be usable in `init` or some sort of guarded `mint`.